src/AutoRest.CSharp/Common/Output/Models/Serialization/ObjectTypeSerialization.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace AutoRest.CSharp.Output.Models.Serialization
 {
-    internal record ObjectTypeSerialization
+    internal class ObjectTypeSerialization
```

Comment:
what is this change for?

Reply:
Pulled from this PR: https://github.com/Azure/autorest.csharp/pull/4795
fixing stack overflow issue during debugging

Reply:
since the above PR has been merged. This change should disappear from this PR